versions.json

```diff
@@ -823,6 +823,24 @@
         ],
         "fpm_directories": ["include","lib"]
     },
+    "jsoncons": {
+        "git_repository": "https://github.com/danielaparker/jsoncons.git",
+        "commitish": "v0.178.0",
+        "version_string": "0.178.0",
+        "license": "BSL-1.0",
+        "consortium_build_number": "0",
+        "package_revision": "0",
+        "externals_root": "opt/irods-externals",
+        "build_steps": [
+            "mkdir -p build",
+            "cd build; TEMPLATE_CMAKE_EXECUTABLE .. -G'Unix Makefiles' -DCMAKE_USER_MAKE_RULES_OVERRIDE=TEMPLATE_SCRIPT_PATH/ClangOverrides.txt -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=TEMPLATE_INSTALL_PREFIX -DCMAKE_INSTALL_LIBDIR=lib -DJSONCONS_BUILD_TESTS=OFF",
+            "cd build; make -jTEMPLATE_JOBS; make install"
```

Comment:
What does the compiled version give us?

The README doesn't appear to say anything about that. Or perhaps this about capturing the directory structures produced by `make install`.

Reply:
There's no compilation, actually. In CMake it's a header-only library. The build steps here are for generating the CMake package configuration.

Reply:
Got it.